common/api/summary/Julius.exports.csv

```diff
@@ -0,0 +1,823 @@
+sep=;
```

Comment:
Where did this file come from?

Reply:
This got added after I ran rush extract-api, I'm not really sure

Reply:
I removed the file for now, but if I run the rush extract-api command it gets added back for some reason which is a bit weird.